webrtc.html

```diff
@@ -1862,6 +1868,39 @@
         </dl>
       </section>
     </section>
+    <section>
+      <h3>Session Negotiation Model</h3>
+      <p>Many changes to the PeerConnection state will require communication with
+        the remote side via the signaling channel. The app can be kept informed
+        as to when it needs to do signaling by listening to the
+        negotiationneeded event.</p>
+      <p>If an operation is performed on a RTCPeerConnection that requires signaling,
+        the connection will be marked as needing negotiation. Examples of such
```

Comment:
In my thinking this was not limited be fired only as a result of an operation the application does, it could be things happening under the hood happening (such as a resource not being available for use any longer, cpu/power issues forcing a re-negotiation etc). This was the main motivator for the event to me - there is no way the app would know about this otherwise.
Of course this would give reason for the follow up question: what if a re-negotiation would be good, but not needed (for example if a HW codec has become available)?